src/main/java/hw01/Pet.java

```diff
@@ -69,21 +64,32 @@ public Integer getTrickLevel() {
     }
 
     public void setTrickLevel(Integer trickLevel) {
-        this.trickLevel = trickLevel;
+        if (trickLevel == null) this.trickLevel = null;
+        else if (trickLevel >= 1 && trickLevel <= 100) {
+            this.trickLevel = trickLevel;
+        } else {
+            throw new IllegalArgumentException("TrickLevel must be between 1 and 100, or null.");
+        }
     }
 
-    @Override
     public void eat() {
         System.out.println("I am eating");
     }
 
-    @Override
     public void respond() {
         System.out.printf("Hello, owner. I am %s. I miss you!" + nickname);
     }
 
-    @Override
     public void foul() {
         System.out.println("I need to cover it up");
     }
+
+    @Override
+    public String toString() {
+        return "Pet habits=" + Arrays.toString(habits) +
```

Comment:
pls fix toString method should be like
`pet=dog{nickname='Rock', age=5, trickLevel=75, habits=[eat, drink, sleep]}}`